Formula/kfctl.rb

```diff
@@ -0,0 +1,28 @@
+class Kfctl < Formula
+  desc "Kubeflow command-line interface"
+  homepage "https://github.com/kubeflow/kfctl"
+  url "https://github.com/kubeflow/kfctl.git",
+      tag: "v1.1.0",
+      revision: "9a3621e2b2ce6d2baa5c78acccec5028c4e4cbe1"
+  license "Apache-2.0"
+
+  depends_on "go@1.13" => :build
```

Comment:
Does it not work with the most recent go?

Reply:
It doesn't work. https://github.com/kubeflow/kfctl/blob/master/Makefile#L16

Reply:
That doesn't mean it won't work with a newer version, but it does make me hesitant to accept it into homebrew, since apparently it's not trying to stay compatible with new releases, which in turn will cause problems for homebrew down the line. In februari this formula will break at this rate.

Reply:
I have tried compiling it in the latest version and it has an issue and that is the reason I had to fall back to `1.13`

Reply:
So do you expect the upstream to fix this before coming with a brew formula?

Reply:
> So do you expect the upstream to fix this before coming with a brew formula?

I expect upstream to not be a year behind in security updates. If there is no work being done to keep up to date I'd consider it abandoned and violating the rule:

https://docs.brew.sh/Acceptable-Formulae#niche-or-self-submitted-stuff
> be maintained (i.e. the last release wasn’t ages ago, it works without patching on all supported macOS releases and has no outstanding, unpatched security vulnerabilities)

Reply:
I agree, closing it waiting for the upstream.